### DISCUSSION TITLE
How can i render a input numeric ?

### DISCUSSION THREAD

**Discussion by ccorreia64** [heroui-inc/heroui#1734] [Q&A] [answered]
Hi!

`<Input size='sm' type='numeric' label='' defaultValue={'2'} />;`

That code does not show the input in numerical format.

Is it possible to use the input component as numeric ?

**Comment by robertostorino** [comment, marked as answer]
Hi!

For example, if you're using NextUI + React-Hook-Form, then you could set valueAsNumber on true at the register section.

 <input type='number' {...register('numeroDeDocumento', { valueAsNumber: true })} />
